src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/ApprenticeForExistingEmployee.cshtml

```diff
@@ -0,0 +1,69 @@
+﻿@using SFA.DAS.EmployerAccounts.Web.Helpers
+@{ViewBag.PageID = "apprentice-for-existing-employee"; }
+@{ViewBag.Title = "Are you setting up an apprenticeship for an existing employee?"; }
+@{ViewBag.MetaDesc = "Are you setting up an apprenticeship for an existing employee"; }
+@{Layout = "~/Views/Shared/_Layout_CDN.cshtml"; }
+
+@{
+    ViewBag.GaData.Vpv = "/account/setupapprentice/apprentice-for-existing-employee";
+    var inError = Model?.GetType().GetProperty("InError") != null;
+}
+
+@section breadcrumb
+{
+    <a href="javascript:history.back()" class="govuk-back-link js-show">Back</a>    
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        @if (inError)
```

Comment:
I think we should be using existing validation controls.

Reply:
Done